modules/bridge/src/main/ts/ephox/bridge/components/content/Autocompleter.ts

```diff
@@ -30,7 +30,9 @@ export interface AutocompleterItem {
 
 export interface AutocompleterSpec {
   type?: 'autocompleter';
-  ch: string;
+  // TODO: TINY-8929: Remove 'trigger' fallback to 'ch'
+  ch?: string;
+  trigger?: string; // TODO: TINY-8929: upgrade to non-optional
```

Comment:
Why is this after when all the other comments are above? This also feels quite verbose, I think we could just merge it in with the first TODO.